.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
   - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6-dev"
```

Comment:
Travis doesn't support the stable branch of 3.6 as of yet ([docs](https://docs.travis-ci.com/user/languages/python/#Choosing-Python-versions-to-test-against)). The argument could be made that we should wait, which is fair, but I still think it's worth putting up this PR to get a run of the tests in travis's environment. Personally I think it is worth merging this in because we don't know how long it will take for Travis to update, but I'd like to hear what others think.